### PR TITLE
New Grant Proposal: Xcavate - the future of real estate finance & pro…

### PR DESCRIPTION
**Pull request by xcavate2022**
#### Project Abstract

> Please replace these instructions with a brief description of your project summarising key points (1-2 paragraphs).
>
> If your application is a follow-up to a previous grant, please mention which one in the first line of the abstract and include a link to previous pull requests if applicable.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [ ] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/grant_guidelines_per_category.md).
- [ ] A BTC, Ethereum (USDT/USDC/DAI) or Polkadot/Kusama (aUSD) address for the payment of the milestones is provided inside the application.
- [ ] The software delivered for this grant will be released under an open-source license specified in the application.
- [ ] The initial PR contains only one commit (squash and force-push if needed).
- [ ] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Comment by semuelle** [comment]
Thanks for the application, @xcavate2022. Someone will look into it more closely soon. In the meantime, could you

- sign the CLA,
- host the images elsewhere? We prefer to keep the repo to the documents only. Easiest is to just paste the images into the Github editor.
- Can you remove the application text from the PR and replace it with our filled out [PR template](https://github.com/w3f/Grants-Program/blob/master/.github/pull_request_template.md)?
- What does "Level: 1 = $10000" mean? At 300k, this is a Level 3 grant application.

**Comment by xcavate2022** [comment]
Thanks for your comments Sebastian

Can you please clarify where the CLA is?
done
done
I was told that we can apply for the initial grant then reapply for more if required… or would you just recommend we adjust and go direct to level 3?

Best Regards,

Richard

**Comment by semuelle** [comment]
> Can you please clarify where the CLA is? 

Follow [the link](https://cla-assistant.io/w3f/Grants-Program?pullRequest=1266) posted by the CLAAssistant in the comment above. All commit authors need to sign it. 

> I was told that we can apply for the initial grant then reapply for more if required… or would you just recommend we adjust and go direct to level 3?

Absolutely. However, your application currently lists a total cost of USD 307,200 and milestones with higher cost than 10k. So it is not clear what work the 10k would go towards. If these are the total costs of your project but you only want to apply for a small part of it, please write the application so that it is clear what it is you want to deliver as part of the grant. 

Given the size of the project you are describing, please also keep in mind that all work funded by a W3F grant needs to be open source and useful for the ecosystem as a whole, so no proprietary dependencies.

File: Project_Xcavate.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+- **Full-Time Equivalent (FTE):**  3
+- **Total Costs:** $36000 USD
+
+### Milestone 1 - Technical & Visual Design and setup
+
+- **Estimated duration:** 1 month
+- **FTE:**  3
+- **Costs:** 36,000 USD
+
+We will start with the user onboarding process to Xcavate parachain/parathread allowing the design team to create low-fi wireframe mockups using Figma that will then be approved by the team to be updated to hi-def mockups that will be interactive for the team's final feedback. This phase will be constant, but the core of the design deliverables will take 3-4 weeks based on feedback. As each of these designs is approved, we will pass them on to the development team to complete in tandem. We will, however, separate these into another milestone. Also, during this first phase, developers will set up the developer's environment with dev env and third-party sandboxes.
```
Comment (semuelle) [inline Project_Xcavate.md:148]: > that will then be approved by the team

Which team? We usually expect at least low-fi designs or wireframes to exist for a UI grant.

File: Project_Xcavate.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
+
+- **Project Name:** Xcavate
+- **Team Name:** Xcavate
+- **Payment Address:** 4pjePvJgUZAMKtnSmYJitGacxBRnsKe1GvCb1t6vgg2Kof8o 
+- **Level:** 1 = $10000
```
Comment (semuelle) [inline Project_Xcavate.md:5]: At the moment, M1 is stated to cost 36,000 USD, so it'd be a Level 3 grant.

File: Project_Xcavate.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+The system will significantly reduce the time from real estate planning permission to build start. By simplifying the finance application process and speeding up the release of funds to the developer.. Xcavate not only provides real-world property developers with a way to build quicker but, as a result, enables a more significant number of much-needed houses to be delivered to market. It also provides liquidity to static real-world assets at every stage of the property development process in a fair and risk-averse way. It will enable the real estate to be onboarded cleanly and efficiently, opening up opportunities throughout the broader Dotsama ecosystem.
+
+![](https://xcavate.io/wp-content/uploads/2022/11/XCAV-WEB-3-Grant-Application-Image-1.jpg)
+
+![](https://xcavate.io/wp-content/uploads/2022/11/XCAV-WEB-3-Grant-Application-Image-2.jpg)
+
+![](https://xcavate.io/wp-content/uploads/2022/11/XCAV-WEB-3-Grant-Application-Image-3.jpg)
```
Comment (semuelle) [inline Project_Xcavate.md:23]: Only two of these images load for me. Interestingly, all the xcavate.io URLs return 404 for me.

File: Project_Xcavate.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+Decentralized Identity Document pallet
+User Management pallet
+Loan Management pallet
+Transaction management pallet
+Staking pallet
+Verification pallet
+
+This Architecture diagram shows the visual representation that maps out the physical implementation for components in the Polkadot ecosystem and how the scalability of the platform works. The diagram represents Microservices we are using to execute the functionality. API gateway will split the API calls to the respective microservices. We are using an s3 bucket to store the media files. We are using pallets to execute the functions on the Polkadot ecosystem.
+
+![](https://xcavate.io/wp-content/uploads/2022/11/XCAV-WEB-3-Grant-Application-Image-4.jpg)
```
Comment (semuelle) [inline Project_Xcavate.md:43]: This image also doesn't load for me.

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the update. Sorry for the delay here. I think @semuelle missed the update. In the future, feel free to ping us here. Regarding your current proposal, it looks like you only want to create wireframes and mock-ups as part of this grant. This is usually something we don’t support since it’s not necessarily useful for others and is very much focused on your product. My recommendation would be to do this work on your own and reapply later once you actually want to work on the open-source software implementation.

**Comment by Noc2** [comment]
I’m closing this for now. Let me know if you want to re-open it.

**Pull request closed without merging**